aws/resource_aws_db_instance_test.go

```diff
@@ -1046,6 +1070,135 @@ resource "aws_db_instance" "snapshot" {
 }`, acctest.RandInt())
 }
 
+func testAccSnapshotInstanceConfigWithS3Import(bucketName string, prefix string, role string) string {
+	return fmt.Sprintf(`
+
+resource "aws_s3_bucket" "xtrabackup" {
+  bucket = "%s"
+}
+
+resource "aws_s3_bucket_object" "xtrabackup_db" {
+  bucket = "${aws_s3_bucket.xtrabackup.id}"
+  key    = "%s/sample.tar.gz"
+  source = "../files/2018-02-26_19-31-02.tar.gz"
```

Comment:
Can we please rename the backup file to something meaningful? e.g. `mysql-5-6-xtrabackup.tar.gz`